pkg/storage/driver/driver.go

```diff
@@ -39,44 +39,44 @@ type Creator interface {
 	Create(key string, rls *rspb.Release) error
 }
 
-// Updator is the interface that wraps the Update method.
+// Updater is the interface that wraps the Update method.
 //
 // Update updates an existing release or returns
 // ErrReleaseNotFound if the release does not exist.
-type Updator interface {
+type Updater interface {
 	Update(key string, rls *rspb.Release) error
 }
 
-// Deletor is the interface that wraps the Delete method.
+// Deleter is the interface that wraps the Delete method.
 //
 // Delete deletes the release named by key or returns
 // ErrReleaseNotFound if the release does not exist.
-type Deletor interface {
+type Deleter interface {
 	Delete(key string) (*rspb.Release, error)
 }
 
-// Queryor is the interface that wraps the Get and List methods.
+// Querier is the interface that wraps the Get and List methods.
 //
 // Get returns the release named by key or returns ErrReleaseNotFound
 // if the release does not exist.
 //
 // List returns the set of all releases that satisfy the filter predicate.
 //
 // Query returns the set of all releases that match the provided label set.
-type Queryor interface {
+type Querier interface {
```

Comment:
This is a breaking change to a public API. Can you please revert this?

Reply:
Sure

Reply:
I presume this applies to the entire family (Updat*/Delet*/Quer*)